st2api/Makefile

```diff
@@ -59,7 +59,7 @@ all: install
 
 install: wheelhouse changelog
 
-post_install:
+post_install: bdist_wheel
```

Comment:
Sorry, I was talking about the `wheel` Python package - https://pypi.python.org/pypi/wheel.

It used to be that `bdist_wheel` command was only available if `wheel` package was installed locally.